sway/desktop/render.c

```diff
@@ -232,6 +234,10 @@ static void render_saved_view(struct sway_view *view,
 		.width = view->saved_buffer_width,
 		.height = view->saved_buffer_height,
 	};
+	struct wlr_box geometry;
+	view_get_geometry(view, &geometry);
+	box.x -= geometry.x;
+	box.y -= geometry.y;
```

Comment:
Shouldn't this use the saved geometry?

Reply:
Yeah, it should. I'll amend it after I get some more feedback.